pandas/core/series.py

```diff
@@ -958,12 +975,19 @@ def to_string(self, buf=None, na_rep='NaN', float_format=None,
         float_format : one-parameter function, optional
             formatter function to apply to columns' elements if they are floats
             default None
-        length : boolean, default False
+        length : boolean, default True
```

Comment:
you didn't actually change the default here (but in any case, this is a good example of why `optional` is better than `default False`)